pkg/registry/minion/rest_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package minion
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+)
+
+func TestMatchNode(t *testing.T) {
```

Comment:
Yeah, for testing that storage objects respect the rest conventions.  It's a bit higher level than this, but see pkg/api/rest/resttest.  


> On Apr 2, 2015, at 4:36 PM, Daniel Smith <notifications@github.com> wrote:
> 
> In pkg/registry/minion/rest_test.go:
> 
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package minion
> > +
> > +import (
> > +	"testing"
> > +
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
> > +)
> > +
> > +func TestMatchNode(t *testing.T) {
> Sorry--I don't follow--is there a fancy new testing package I'm not aware of?
> 
> —
> Reply to this email directly or view it on GitHub.
>